libs/platform/src/lib/components/form/form-group/form-field/form-field.component.ts

```diff
@@ -69,19 +69,15 @@ export class FormFieldComponent implements FormField, AfterContentInit, AfterVie
     @Input()
     noLabelLayout = false;
 
+    @Input()
+    validators: Array<ValidatorFn> = [Validators.nullValidator];
+
```

Comment:
In general all these Inputs should have typedocs as that's how the API is built in the documentation.